### ISSUE TITLE
Test data regression w/deserializing custom objects

### ISSUE THREAD

**Issue by asmaloney** [CloudCompare/CloudCompare#798]
## Current Behavior
The `Mess.bin` and `Mess2.bin` files from the test data set fail to load.

## Expected Behavior
They should load.

## Steps to Reproduce (for bugs)
Try loading them :-)

```
[06:14:48] [ccHObject::fromFile] Couldn't found any plugin able to deserialize custom object 'Quadric (Mesh_group_big_coords_no_texture.sampled.remaining.remaining.clone)' (class_ID = 8589935241 / plugin_ID = 
```

## Context/Comments
This is a result of change c25e1a84ff9898a2fa919438b0aa820a00cf2122.

## Your Environment
* CC Version: git master

**Comment by dgirardeau** [comment]
No the error is actually that the CC_CAMERA_BIT and CC_QUADRIC_BIT are wrongly defined :D. They have 2 bits defined instead of 1, and 1 bit is the CC_CUSTOM_BIT one!

(it's like this since 2014?! Apparently it appeared when the intern who created the Camera sensor defined the CC_CAMERA_BIT bit... oops... Then the Quadric one was copy-pasted from the CC_CAMERA_BIT :( ).

**Comment by dgirardeau** [comment]
Okay that's fixed! 85ad8820730b0e60ab2cb16ebeee23d4f0929b5b

Thanks again.
